app/commands/get_review_recommendations.rb

```diff
@@ -1,4 +1,5 @@
-class GetReviewRecommendations < PowerTypes::Command.new(:github_user_id, :other_users_ids)
+class GetReviewRecommendations < PowerTypes::Command.new(:github_user_id, :other_users_ids,
+                                                         month_limit: nil)
```

Comment:
Use one level of indentation for parameters following the first line of a multi-line method call.